frontend/src/app/app.module.ts

```diff
@@ -38,10 +41,14 @@ export function HLLanguages() {
         ReactiveFormsModule,
         HighlightModule.forRoot({
             languages: HLLanguages
-        })
+        }),
+        MaterialModule,
+        FormsModule,
+        HttpClientModule
```

Comment:
45, 46, 47 and 38, 39, 40 lines are duplicated. it makes such a mistake " error TS2300: Duplicate identifier 'FormsModule'" when run project